routes/profiles.js

```diff
@@ -48,10 +48,8 @@ router.get(
 
     let extensions = profileUser.Extensions
     let comments = profileUser.Comments
-    // console.log(profileUser.Extensions[0].name)
-    // console.log(profileUser.Comments[0].content)
 
-    res.render('profile-page', { title: "Profile Page", profileUser, req, extensions, comments, csrfToken: req.csrfToken() })
+    res.render('profile-page', { title: "Profile Page", profileUser, req, extensions, comments, csrfToken: req.csrfToken(), authenticated: !!res?.locals.authenticated })
```

Comment:
I'm not sure how to read this line regarding the authenticated. I am assuming it returns authenticated as a boolean that is then read in the profile-page.pug. I guess I just don't know that syntax.

Reply:
`res?.locals.authenticated` returns undefined instead of crashing if `res.locals` is undefined. `!!undefined` converts to `false`